x-pack/plugins/security_solution/scripts/endpoint/resolver_generator_script.ts

```diff
@@ -360,7 +381,8 @@ async function main() {
       ancestryArraySize: argv.ancestryArraySize,
       eventsDataStream: EndpointDocGenerator.createDataStreamFromIndex(argv.eventIndex),
       alertsDataStream: EndpointDocGenerator.createDataStreamFromIndex(argv.alertIndex),
-    }
+    },
+    DocGenerator
```

Comment:
Should I add an additional CLI param to revert back to prior behaviour?

Reply:
Maybe a flag to generate mixed version data that defaults to false?

Reply:
Otherwise generate current version only

Reply:
cool. I'll add it.
I'm also going to see if I can get the kibana version from API agains the running kibana and not pull it from the `package.json` - since we sometime use this tool to load data into cloud env.'s for testing